docs/agent-vs-gateway.md

```diff
@@ -1,3 +1,5 @@
+> The official Splunk documentation for this page is [Determine your deployment mode](https://docs.splunk.com/Observability/gdi/opentelemetry/deployment-modes.html). For instructions on how to contribute to the docs, see [CONTRIBUTE.md](../CONTRIBUTE.md).
```

Comment:
```suggestion
> The official Splunk documentation for this page is [Determine your deployment mode](https://docs.splunk.com/Observability/gdi/opentelemetry/deployment-modes.html). For instructions on how to contribute to the docs, see [CONTRIBUTE.md](../CONTRIBUTING.md#documentation).
```

Reply:
All links fixed, thanks for spotting! 🤦🏻